### DISCUSSION TITLE
Publish linux_aarch64 binary for Graviton

### DISCUSSION THREAD

**Discussion by ericsun2** [qdrant/qdrant#3041] [General]
Qdrant has binary for Apple drawin_aarch64 already, does this work for AWS Graviton? Since it is RUST, I am not sure these 2 ARM-based OS are fully compatible in terms of Qdrant.

It will be great to have `linux_aarch64` binary. 🙇

**Comment by timvisee** [comment]
We don't ship an `aarch64-unknown-linux-gnu` targeted raw binary at this time as part of our releases.

However, we do [provide](https://hub.docker.com/r/qdrant/qdrant/tags) Docker images for arm64 which should run perfectly fine on AWS Graviton. This is also our recommended way of [running](https://qdrant.tech/documentation/quick-start/#download-and-run) Qdrant.

If you'd like raw binaries for this platform you can definitely compile Qdrant yourself. It's done through Rust's `cargo` and is quite simple to do.
